### ISSUE TITLE
Service Recommendation Fields for LWC

### ISSUE THREAD

**Issue by AIrwin33** [Salesforce-org-Impact-Labs/01HousingandHomelessness#12]
Service
-Id
-Provider Name
-Program Name
-Program Type (Food, Housing, Job Training, etc. ) 
 Should these categories be determined by the service provider during configuration?
-Program Description
-Hours for that day (ex. Closed on Mondays, Open until 5:00 PM)
-Street
-City
-Phone
-Website
-Upvotes
-Downvotes
-Comments
-List/Map of Relevant Indicators to each indicator value for relevance?

<img width="453" alt="Screen Shot 2020-06-04 at 7 33 10 PM" src="https://user-images.githubusercontent.com/9088679/83823726-3d30bc80-a69a-11ea-86b0-69ac2352489c.png">

**Comment by mshanemc** [comment]
> -Program Type (Food, Housing, Job Training, etc. )
Should these categories be determined by the service provider during configuration?

Like, you'd want to include/exclude at these category levels?  If the aggregators are fairly consistent with the categories, we could.

**Comment by AIrwin33** [comment]
Yeah, Like "only show me these categories of services". Would that still be possible to do that kind of filtering?

At the very least, I would like to have a small amount (less than 10) of program categories across all programs returned since that is going to drive the icon we use in the LWC.

**Comment by mshanemc** [comment]
also, the comment feature is removed
https://docs.google.com/presentation/d/1YCmSxAXPgg4th19g5BKmDzMEsJM45qk0t69bI6pj2Oc/edit#slide=id.g872c3fe9e9_0_94

**Comment by mshanemc** [comment]
@AIrwin33 if we were storing the service categories, we could (it'd be easy to pull the list of categories, and store a include/exclude list)

But if some aggregators don't let us store that info, then you'd have to know in advance what the possible values are.  Ask Tim about it

**Comment by tweeksPHL** [comment]
AB which uses Open Eligibility standards uses 10 high level categories to describe services. Those could be a good starting point. While service tags might vary across aggregators, I think we could map most into this structure for future integrations. Thoughts?

- Food
- Housing
- Goods
- Transit
- Health
- Money
- Care
- Education
- Work
- Legal

**Comment by mshanemc** [comment]
Let's go with that

**Comment by AIrwin33** [comment]
Thanks @mshanemc @tweeksPHL

**Comment by AIrwin33** [comment]
@tweeksPHL Do you know how hours at the service are sent to us from AB? From the UX we're only showing how late the service is open that day (open until 4:00 PM), or if the service is closed (Closed Fridays).

**Comment by mshanemc** [comment]
AB has a boolean for each day (is it open at all?), and then a start/end hour for that day

I'll have a related object with start/end and picklist for day

**Comment by mshanemc** [comment]
you should have all that, with sample data for accounts, services, hours now.

I also added an external_id field on Service so we can tie it to the Aggregator (either to update stuff OR when AB won't let us store data and we have to match in-org storage vs. real-time call results)
